fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/Payload/A320_251N/A320Payload.tsx

```diff
@@ -58,6 +59,8 @@ export const A320Payload: React.FC<A320Props> = ({
     const [cFlagsDesired, setCFlagsDesired] = useSeatFlags(`L:${Loadsheet.seatMap[2].simVar}_DESIRED`, Loadsheet.seatMap[2].capacity, 457);
     const [dFlagsDesired, setDFlagsDesired] = useSeatFlags(`L:${Loadsheet.seatMap[3].simVar}_DESIRED`, Loadsheet.seatMap[3].capacity, 499);
 
+    const globalSettingsRedux = useSelector((state:any) => state.globalSettings);
```

Comment:
In theory you should avoid using `any`, and you can do so here by calling `useAppSelector()` instead from `instruments/src/EFB/Store/store`:
```ts
export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
```

This means that when you call `useAppSelector()`, all the reducers are present since `RootState` is being passed in.

As a result, you're able to change this to:

```suggestion
    const globalSettingsRedux = useAppSelector((state) => state.globalSettings);
```